module/NPC-sheet.js

```diff
@@ -233,19 +233,19 @@ activateListeners(html) {
       let brackets = "";
       // let description = "";
       let description = "<hr style='margin:3px 0;'><img class='description-image-chat' src='" + item.img + "' width='50' height='50'/>" + item.data.description;
-      let points = " points";
+      let points = " " + game.i18n.localize("CYPHERSYSTEM.Points");
```

Comment:
I’m not sure if that’s easily changed, but for consistency’s sake, it’s recommended to use single quotes for the localization string (i.e. `game.i18n.localize('CYPHERSYSTEM.Points')`) in order to have it more easily converted between HTML handlebars and JS code.

But that’s nice just to have if it’s easily converted, not a necessity.

Reply:
It should not be a problem. But when I was looking at others translated system the single quote was only when using the `{{localize 'CYPHERSYSTEM.Points' }}` and not the `game.i18n.localize('CYPHERSYSTEM.Points')`.
Will change it either way.

Reply:
Before you do, test if it makes a difference. I think I mis-remembered the advice given here: [Languages and Localization](https://foundryvtt.com/article/localization/)

Reply:
Apparently it is ' for HTML and " for JS:
![image](https://user-images.githubusercontent.com/38940602/112154252-5e8c7600-8be4-11eb-9957-04cd72eb01db.png)
![image](https://user-images.githubusercontent.com/38940602/112154272-63e9c080-8be4-11eb-8060-7a37edfb3c72.png)

So all good!